ui-components/src/locales/general.json

```diff
@@ -390,6 +390,10 @@
           "label": "Displaced Tenant Housing Preference",
           "description": "Displaced Tenant copy goes here…",
           "link": "http://domain.com"
+        },
+        "missionCorridor": {
+          "label": "Mission Corridor",
+          "description": "Mission Corridor copy goes here…"
```

Comment:
Is this not available yet?

Reply:
Nice catch! It's a new version of translation with proper structure, I'll remove old positions.